docs/src/configuration/app/cron.md

```diff
@@ -23,6 +24,44 @@ If an application defines both a `web` instance and a `worker` instance, cron ta
 Cron log output is captured in the at `/var/log/cron.log`.  See the [Log page]({{< relref "/development/logs.md" >}}) for more information on logging.
 {{< /note >}}
 
+### Commands
+
+There are two mutually-exclusive ways to define the cron commands.
+
+The full version is the following:
+
+```yaml
+crons:
+    mycommand:
+        spec: '*/17 * * * *'
+        commands:
+            start: <command here>
+            stop: <command here>
```

Comment:
I feel like this version is not making it obvious that this field and the shutdown_timeout fields are optional

Reply:
@Crell I’m thinking either

a) 
```
crons:
    mycommand:
        spec: '*/17 * * * *'
        commands:
            start: <required; command here>
            stop: <command here>
            shutdown_timeout: <integer here, default 10s>
```
b) or splitting the next paragraphs:

* **`start` (Required):** The `start` key is the cron command that should execute according to the specified schedule.  It is required, and is often all you need.
* **`stop` (Optional):** If a cron task is interrupted by a user through the CLI or Web Admin Console, then the `stop` command will be issued to give the cron command a chance to shutdown gracefully, such as finish an active item in a list of tasks.  The `stop` command is optional, and if not specified then a `SIGTERM` signal will be sent to the process.
* **`shutdown_timeout` (Optional):** If the cron process is still running after `shutdown_timeout` seconds, a `SIGKILL` signal will be sent to the process to force terminate it.  If not specified, the default `shutdown_timeout` is 10 seconds.

b is probably more clear

Reply:
I went with B, using the formatting convention already in place on the main app.yaml page.